### DISCUSSION TITLE
Table Record Selectable one at a time

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#6955] [Help]
Is it possible that the option to select records from a table can be selected only one at a time, which does not allow multiple selection of records from the table?

**Comment by danharrin** [comment]
Whats the point? We have row-level actions for one-at-a-time operations

**Reply by Hexml-dev** [reply]
If you have a modal with a table in it and wish to pick one entry then close the modal...

**Reply by danharrin** [reply]
We have this feature now, `maxSelectableRecords(1)`
